src/auth/authentication-provider/affinidi-authentication-provider.ts

```diff
@@ -194,10 +228,18 @@ export class AffinidiAuthenticationProvider implements AuthenticationProvider, D
     const removed = []
     if (oldSession) removed.push(oldSession)
     if (newSession) added.push(newSession)
+    if (newSession) configVaultService.setCurrentUserID(newSession.account.id)
     this._onDidChangeSessions.fire({
       added,
       removed,
       changed: [],
     })
   }
+
+  handleExternalChangeActiveProject = (): void => {
+    this._onDidChangeActiveProject.fire({
+      type: FileChangeType.Changed,
+      uri: Uri.file(path.join(os.homedir(), '.affinidi/config.json')),
```

Comment:
Why do we need URI on active project events? Shouldn't this be abstracted away by the Credentials Vault?

Reply:
we are keeping active project id in config file, I am watching changes in config file, after that also updating the credentials file

Reply:
understood what you meant actually, removed the fire method here and used setActiveProject